main.go

```diff
@@ -101,18 +102,27 @@ func handleUfwRule(ch <-chan ufwEvent) {
 	}
 }
 
-func main() {
-	ctx := context.Background()
+func createClient() *client.Client {
 	cli, err := client.NewClientWithOpts(client.FromEnv)
 	if err != nil {
 		panic(err)
 	}
+	return cli
+}
 
+func addFilters(cli *client.Client, ctx *context.Context) (<-chan events.Message, <-chan error) {
+	_, cancelContext := context.WithCancel(*ctx)
+	cancelContext()
```

Comment:
Could you tell me what it does? :)

Reply:
Cancels context, and therefore any go channels might be cleared I guess, let me know if I am wrong here, also I will remove it you think there is no need of this.

Reply:
Yeah, I'm not that familiar with Go contexts... For now best feedback I can give is as long as it works correct ... we'll keep it.

Reply:
Agreed